config/routes.rb

```diff
@@ -9,12 +9,13 @@
   end
 
   namespace :admin do
-    resources :rebate_forms, only: %i[show index update destroy]
+    resources :rebate_forms, only: %i[show index update destroy edit post]
     resources :attachments, only: %i[destroy]
     resources :councils
     resources :users
     resources :batches
     get 'signature' => 'signatures#show'
+    post "rebate_forms/:id/edit" => "rebate_forms#edit"
```

Comment:
the `resources` route will make an edit for you.
have a look at the output of `rake routes`

Reply:
so , you shouldn't need this line at all